Userland/Services/WebDriver/Session.cpp

```diff
@@ -414,6 +414,58 @@ ErrorOr<JsonValue, HttpError> Session::find_element(JsonValue const& payload)
     return JsonValue(result.at(0));
 }
 
+// 12.3.3 Find Elements, https://w3c.github.io/webdriver/#dfn-find-elements
+ErrorOr<JsonValue, HttpError> Session::find_elements(JsonValue const& payload)
+{
+    if (!payload.is_object())
+        return HttpError { 400, "invalid argument", "Payload is not a JSON object" };
+
+    auto const& properties = payload.as_object();
+    // 1. Let location strategy be the result of getting a property called "using".
+    if (!properties.has("using"sv))
+        return HttpError { 400, "invalid argument", "No property called 'using' present" };
+    auto const& maybe_location_strategy = properties.get("using"sv);
+    if (!maybe_location_strategy.is_string())
+        return HttpError { 400, "invalid argument", "Property 'using' is not a String" };
```

Comment:
We really need some helper functions for typed property lookup, this is getting out of hands lol